### ISSUE TITLE
plainButtonWithNormalImageName method not found

### ISSUE THREAD

**Issue by edison0951** [Instrument/Vicinity#10]
First, I download the code, and then I run the code, the error say the EasyLayout.h not found. so I download the EasyLayout Code, and add it to the Xcode project. And I run again, the error say plainButtonWithNormalImageName not found.I have to annotate the code in order to run successfully. where is the method?

**Comment by bentford** [comment]
Sounds like the submodules are missing.  Use `git clone --recursive` or `git submodule init; git submodule init;`

**Comment by wcchh** [comment]
So, any update about "plainButtonWithNormalImageName"? since there's no where to find it~

**Comment by bentford** [comment]
The `plainButtonWithNormalImageName` is in the EasyLayout project. 
https://github.com/bentford/EasyLayout/blob/master/EasyLayoutLibrary/ButtonMaker.h

Make sure to checkout submodules using command listed above.

**Comment by wcchh** [comment]
Yes, but I only saw "plainButtonWithNormalImage", not "plainButtonWithNormalImageName"; Is that right?
Sorry for I'm new in iOS, please let me know if I'm wrong, thanks.

**Comment by jkroeckel** [comment]
To fix, in FindDeviceViewController.m, change to this line:

```
modeButton = [ButtonMaker plainButtonWithNormalImage: 
                                [UIImage imageNamed:@"mode_button_detecting.png"]
                                selectedImage:[UIImage imageNamed:@"mode_button_broadcasting.png"]];
```
